### PR TITLE
BOT-1875 prod bug test

### PR DESCRIPTION
**Pull request by thivya-deriv**
Card: https://app.clickup.com/t/20696747/BOT-1931
Desc: Test for production bug BOT-1875 and fix flaky error

### CODE REVIEW

**Review by bilalmajid-deriv — APPROVED** [review]

File: cypress/e2e/smoke/dbot/QuickStrategies.cy.js [hunk 1]
```diff
@@ -52,6 +52,18 @@ describe('QATEST-4140: Verify Dashboard [Mobile]', () => {
...
+      if (isMobile) cy.get('.dc-drawer__toggle').click({ force: true })
+      quickStrategy.clickQuickStrategies()
+      cy.findByTestId('dt_qs_tradetype')
+        .invoke('val')
+        .then((value) => {
+          cy.log(`Trade type ${value} is visible`)
+          cy.findByTestId('dt_qs_durationtype')
+            .invoke('val')
+            .then((value2) => {
+              cy.log(`Duration ${value2} is visible`)
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/dbot/QuickStrategies.cy.js:64]: if(!value) , fail the test.

File: cypress/e2e/smoke/dbot/QuickStrategies.cy.js [hunk 2]
```diff
@@ -52,6 +52,22 @@ describe('QATEST-4140: Verify Dashboard [Mobile]', () => {
         })
         runPanel.transactionAfterFirstLoss()
       }
+      if (isMobile) cy.get('.dc-drawer__toggle').click({ force: true })
+      quickStrategy.clickQuickStrategies()
+      cy.wait(7000)
```
Comment (meeyun-deriv) [inline cypress/e2e/smoke/dbot/QuickStrategies.cy.js:57]: why the wait?

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]
the test seems missing the winning scenario - where the stake will use the same amount

File: cypress/e2e/smoke/dbot/QuickStrategies.cy.js (hunk 2 above)
Comment (thivya-deriv) [reply cypress/e2e/smoke/dbot/QuickStrategies.cy.js]: Fixed now

File: cypress/e2e/smoke/dbot/QuickStrategies.cy.js (hunk 1 above)
Comment (thivya-deriv) [reply cypress/e2e/smoke/dbot/QuickStrategies.cy.js]: This is done

File: cypress/e2e/smoke/dbot/QuickStrategies.cy.js [hunk 3]
```diff
@@ -52,6 +52,20 @@ describe('QATEST-4140: Verify Dashboard [Mobile]', () => {
         })
         runPanel.transactionAfterFirstLoss()
       }
+      if (isMobile) cy.get('.dc-drawer__toggle').click({ force: true })
+      quickStrategy.clickQuickStrategies()
+      cy.findByTestId('dt_qs_tradetype')
+        .invoke('val')
+        .then((value) => {
+          cy.log(`Trade type ${value} is visible`)
+          cy.get('[data-testid="dt_qs_durationtype"]')
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/dbot/QuickStrategies.cy.js:61]: Why this; why not?
```suggestion
          cy.findByTestId('dt_qs_durationtype')
```

File: cypress/e2e/smoke/dbot/QuickStrategies.cy.js (hunk 1 above)
Comment (abuzar-deriv) [reply cypress/e2e/smoke/dbot/QuickStrategies.cy.js]: Where? 😅

Comment (thivya-deriv) [reply cypress/e2e/smoke/dbot/QuickStrategies.cy.js]: Please check again now

**Comment by thivya-deriv** [comment]
> the test seems missing the winning scenario - where the stake will use the same amount

This will be done separately in BOT-1973

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**